tests/ftqc/test_pauli_tracker.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2018-2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit tests for the ftqc.pauli_tracker module"""
+
+import pytest
+
+import pennylane as qml
+from pennylane.ftqc import pauli_encode_xz, pauli_prod_to_xz
+
+
+class TestPauliTracker:
+    """Test for the pauli tracker related functions."""
+
+    @pytest.mark.parametrize("op", [qml.I, qml.X, qml.Y, qml.Z, qml.S])
+    def test_pauli_encode_xz(self, op):
```

Comment:
[Optional] Since the expected results are straightforward here, could we parameterize them next to the corresponding gate in the list of test inputs, e.g. something like

```suggestion
    @pytest.mark.parametrize("op, expected", [(qml.I, (0, 0)), (qml.X, (1, 0)), (qml.Y, (1, 1)), (qml.Z, (0, 1)), (qml.S, None)])
    def test_pauli_encode_xz(self, op, expected):
```

Reply:
Done!